src/layouts/index.vue

```diff
@@ -26,6 +26,19 @@ export default Vue.extend({
       version: GIT_REVISION_SHORT,
     };
   },
+  computed: {
+    network(): string {
+      return this.$store.getters["zk-provider/network"];
+    },
+  },
+  watch: {
```

Comment:
...and the watcher, again, IMO, doesn't belong here: when u change the network it's not a state mutation, it's a trigger to cast sequence of actions, including the logout, redirection, re-login etc.

To maintain solidity of an architecture I propose u to:
- add global call though the one-time registration ``` Vue.use()``` of a single method like track() 
- move everything caused by the network switching trigger into a single method
- compose analytics itself into a reusable plugin;
— do the same for the network switcher.

**PROs**
- one place to change in need;
- reusable code;
- keeping logic away from the UI and from the data-processors/controllers)